package.json

```diff
@@ -48,7 +48,7 @@
       },
       {
         "command": "tomcat.server.create",
-        "title": "New Tomcat Server",
+        "title": "Create Tomcat Server",
```

Comment:
Is 'Add' a better choice? To me, this operation is to add a server registration but not as heavy as creating.

Reply:
suggestion taken.